lib/utils.pm

```diff
@@ -2300,7 +2300,7 @@ sub permit_root_ssh_in_sol {
     my $sshd_config_file = shift;
 
     $sshd_config_file //= "/etc/ssh/sshd_config";
-    enter_cmd("[ `grep \"^PermitRootLogin *yes\" $sshd_config_file | wc -l` -gt 0 ] || (echo 'PermitRootLogin yes' >>$sshd_config_file; systemctl restart sshd)");
+    enter_cmd("[ `grep \"^PermitRootLogin *yes\" $sshd_config_file | wc -l` -gt 0 ] || (echo 'PermitRootLogin yes' >>$sshd_config_file; systemctl restart sshd)", wait_still_screen => 5);
```

Comment:
I guess there will be still chances that permit_root_ssh_in_sol fail since typing commands in sol console usually have errors.  It won't be fully solved until we switch to autoyast host installation.  We can prioritize task if it indeed is quite unstable here. Of course not in this PR, but later. HDYT?